Gem/Code/Source/AutoGen/NetworkCharacterComponent.AutoComponent.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkCharacterComponent"
```

Comment:
Is there a delete for CharacterComponent.AutoComponenet.xml or is it counting this a rename?

Reply:
it should just be a rename